docs/use_cases/retrieval_from_image_and_text.md

```diff
@@ -0,0 +1,268 @@
+[OpenClip leaderboard](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_results.csv) and apply the Image Encoder 
+to embed the images, followed by an evaluation of the outcomes.
+
+![](assets/use_cases/retrieval_from_image_and_text/table_multimodal_vit_embed_image.png)
+
+
+The performance of the tested models remained consistent across both datasets. ViT-based models outperformed the 
+[ResNet50](https://arxiv.org/abs/1512.03385)-based model on COCO, while on the Open Images V7 dataset, the difference was not as 
+significant, despite ViT models having more than 4 times as many parameters. Additionally, we present results from BLIP, which 
+utilizes a ViT model for image encoding.
+
+![](assets/use_cases/retrieval_from_image_and_text/table_multimodal_vit_embed_image_blip.png)
+
+BLIP achieved the best MRR scores on both datasets, surpassing the OpenCLIP models, aligning with the findings in the BLIP paper. 
+The larger BLIP model has parameters totaling 447 million, while the base model has 224.7 million parameters. Notably, the largest 
+BLIP model reached MRR scores of 0.494 and 0.112 on COCO and Open Images V7, respectively.
+
+### 5. Embedding both images and their captions with Multimodal Vision Transformers
+
+In this final scenario, we leverage both the Text and Image encoders from the CLIP and BLIP models to encode the captions 
+and images separately, then concatenate these embeddings. A key difference from scenario 3. is that, here, the encoders 
+have undergone joint pre-training in the context of CLIP or have been aligned with additional layers in the case of BLIP.
+
+![](assets/use_cases/retrieval_from_image_and_text/table_multimodal_vit_embed_image_text.png)
+
+The order of the ViT-based OpenCLIP models is interchanged in these experiments, but their performance remains comparable. 
+Overall, the individual model's performance is roughly consistent on both datasets. In the BLIP experiments, these models once 
+again prove to be more efficient, with the largest model reaching an MRR score of 0.4953 and 0.112 on COCO and 
+Open Images V7 datasets, respectively. 
+
+![](assets/use_cases/retrieval_from_image_and_text/table_multimodal_vit_embed_image_text_blip.png)
+
+In comparison to scenario 3, the concatenation of embeddings from two jointly trained or aligned encoders can boost 
+retrieval performance. Upon observation of these datasets, the boost is more pronounced for OpenCLIP models.
+
+### Comparison of All Results
+
+Now, let's put all the results side by side for comparison.
+
+![](assets/use_cases/retrieval_from_image_and_text/table_all_scenarios_coco.png)
+
+![](assets/use_cases/retrieval_from_image_and_text/table_all_scenarios_oiv7.png)
+
+In both the COCO and Open Images V7 datasets, the BLIP and OpenCLIP models emerged as the most efficient feature extractors. 
+On the COCO dataset, the BLIP model achieved comparable performance when utilizing only the image modality. The top 
```

Comment:
```suggestion
On the COCO dataset, the BLIP model achieved comparable performance when utilizing only the image modality. In general it is established that utilising both the image and the captions will only marginally make the higher performing one (either best image-, or text embedding model) perform better. That might be because the caption and the image is referring to the same abstract notion and these models are doing a similarly good job encoding those. The top 
```